token/cli/src/main.rs

```diff
@@ -1891,6 +1911,164 @@ async fn command_sync_native(
     })
 }
 
+// Both enable-required_transfer-mesos and disable-required_transfer-mesos, switches with enable bool
+async fn command_required_transfer_memos(
+    token_account_address: Pubkey,
+    owner: Pubkey,
+    bulk_signers: BulkSigners,
+    enable: bool,
+    config: &Config<'_>,
+) -> CommandResult {
+    let program_id = if config.sign_only {
+        config.program_id
+    } else {
+        config
+            .rpc_client
+            .get_account(&token_account_address)
+            .await
+            .map_err(|err| {
+                format!(
+                    "Token account {} does not exist: {}",
+                    token_account_address, err
+                )
+            })?
+            .owner
+    };
```

Comment:
there are some helpful sanity checks to do here... particularly we like to call `check_owner`, and it would be nice to halt if they are trying to enable when enabled or disable when disabled. this is kind of rote, you can look at the recent commit 38a9e3af72c156cf10b48b979c963c4787e720a3 to get an idea, or i could just handle it

Reply:
owner sanity check resolved in 0eb49c3
halting command resolved in d37c851